lib/domain/repository/movie_repository_interface.dart

```diff
@@ -5,4 +5,5 @@ import '../../data/model/movie_model.dart';
 
 abstract class IMovieRepository {
   Future<DataState<List<MovieModel>>> getMovies({required Endpoint endpoint});
+  Future<DataState<List<MovieModel>>> getFavoriteMovies({required List<int> moviesId});
```

Comment:
Same rename here

Reply:
done. I removed this method due to the refactor implemented